src/domain/registration/form/RegistrationForm.tsx

```diff
@@ -61,18 +60,28 @@ const RegistrationForm: FunctionComponent<Props> = ({
               setFormValues(values);
 
               // FIXME: Ensure that relationship is submitted to backend
-              const backendSupportChildren = values.children.map(child =>
-                omit(child, ['postalCode', 'homeCity', 'relationship'])
-              );
+              const backendSupportChildren = values.children.map(child => {
+                const c = omit(child, [
```

Comment:
you dont need variable c, plus extra unnecessary texts
Remove all the extra, the left side is already fine, no need to change.

Reply:
At the moment we are not able to submit relationship info to the backend, I don't think the left side is fine. But sure, we can find a different way of doing that.

Reply:
I reverted this change. We have to fix this very soon, but in a different PR